ISSUE_TRIAGE.rst

```diff
@@ -0,0 +1,92 @@
+
+This document is a hands-on manual for doing issue and pull request triage for `MLflow issues 
+on GitHub <https://github.com/mlflow/mlflow/issues>`_ . 
+The purpose of triage is to speed up issue management and get community members faster responses. 
+
+Issue and pull request triage has three steps:
+
+- assign one or more process labels (e.g. ``needs design`` or ``help wanted``),
+- mark a priority, abd 
+- label one or more relevant areas, languauges, or integrations to help route issues to appropriate contributors or reviewers.
+
+The remainder of the document describes the labels used in each of these steps and how to apply them.
+
+Assign appropriate process labels
+#######
+Assign at least one process label to every issue you triage. 
+
+- ``needs author feedback``: We need input from the author of the issue or PR to proceed.
+- | ``needs design``: This feature is large or tricky enough that we think it warrants design doc 
+  | and review before someone begins implementation.
+- | ``needs review``: Use this label for issues that need a more detailed design review or pull 
+  | requests ready for review (all questions answered, PR updated if requests have been addressed, 
+  | tests passing).
+- ``help wanted``: We would like community help for this issue.
+- ``good first issue``: This would make a good first issue.
+
+
+Assign priority
+#######
+
+You should assign a priority to each issue you triage. We use `kubernetes-style <https://github.com/
+kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority>`_ priority 
+labels.
+
+- | ``priority/critical-urgent``: This is the highest priority and should be worked on by
+  | somebody right now. This should typically be reserved for things like security bugs, 
+  | regressions, release blockers.
+- | ``priority/important-soon``: The issue is worked on by the community currently or will 
+  | be very soon, ideally in time for the next release.
+- | ``priority/important-longterm``: Important over the long term, but may not be staffed or
+  | may need multiple releases to complete. Also used for things we know are in a 
+  | contributor's roadmap in the next few months. We can use this with ``help wanted`` to mark 
+  | issues we would like to get help from. If someone starts working on it actively and we 
+  | think it could make the next release, change the priority to ``priority/important-soon``.
```

Comment:
```suggestion
- | ``priority/important-longterm``: Important over the long term, but may not be staffed or
  | may need multiple releases to complete. Also used for things we know are on a 
  | contributor's roadmap in the next few months. We can use this in conjunction with 
  | ``help wanted`` to mark issues we would like to get help with. If someone begins actively
  | working on an issue with this label and we think it may be merged by the next release, change
  | the priority to ``priority/important-soon``.
```